### PR TITLE
Theme generator added to the new site

### LINKED ISSUE #1394: Move the Theme Builder to website
Currently, the theme-builder project is in the /docs directory and uses the old URL format. To change this, we need to move the /themeBuilder implementation to the new site /website, decide its location and rooting, and change the .jsx to .tsx.

Also, we need to decide on the navigation between the new site and the theme builder and how to go from one to another, in both directions.

Apart from that, the tool also needs to be updated visually to look consistent with out current documentation site.

### PR DESCRIPTION
**Pull request by GomezIvann**
**Checklist**
- [x] Build process is done without errors and all tests pass in the `/lib` directory.
- [x] Self-reviewed the code before submitting.
- [x] Meets accessibility standards.
- [x] Added/updated documentation to `/website` as needed.
- [x] Added/updated tests as needed.

**Description**
The theme builder has been renewed with a visual update and moved to a new page of the new Halstack site. There's no need to go outside of the documentation anymore.

For the moment, the tests will not be moved to the website since we need to investigate how to launch them. This will be tackled in a different task.

**Screenshots**

image

**Closes #1394**

### CODE REVIEW

**Review by raquelarrojo — CHANGES_REQUESTED** [review]

File: website/screens/theme-generator/components/previews/Card.tsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+const Card = () => (
+  <PreviewContainer>
+    <Mode text="Default">
+      <DxcCard
+        imageSrc="https://picsum.photos/id/1022/200/300"
+        imageCover={true}
+        margin="small"
+        contentPadding="xsmall"
+      >
```
[the remark this answers is not in the document]
Comment (GomezIvann) [reply website/screens/theme-generator/components/previews/Card.tsx:14]: Done!

File: website/screens/theme-generator/components/previews/Dialog.tsx [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+            <DxcFlex alignItems="flex-end" gap="1rem">
+              <DxcButton
+                label="Cancel"
+                onClick={handleClickDefaultDialog}
+                mode="text"
+              />
+              <DxcButton label="Delete" onClick={handleClickDefaultDialog} />
+            </DxcFlex>
+          </DxcDialog>
+        )}
```
[the remark this answers is not in the document]
Comment (GomezIvann) [reply website/screens/theme-generator/components/previews/Dialog.tsx:64]: The second image can be achieved due to the component's restriction. It seems to have an unexpected `padding-top`.

image

File: website/screens/theme-generator/components/previews/Dialog.tsx [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+            <DxcFlex alignItems="flex-end">
+              <DxcButton
+                label="Cancel"
+                onClick={handleClickBackground}
+                mode="text"
+              />
+              <DxcButton label="Add client" onClick={handleClickBackground} />
+            </DxcFlex>
+          </DxcDialog>
+        )}
```
[the remark this answers is not in the document]
Comment (GomezIvann) [reply website/screens/theme-generator/components/previews/Dialog.tsx:89]: Same as above. Fixed what I could.

File: website/screens/theme-generator/themes/advanced-theme.json [hunk 4]
```diff
@@ -0,0 +1,1175 @@
...
+    "thumbFocusColor": "#005FCC",
+    "thumbFocusColorOnDark": "#1682FF",
+    "thumbHeight": "24px",
+    "thumbWidth": "24px",
+    "thumbShift": "1.25rem",
+    "trackHeight": "12px",
+    "trackWidth": "36px",
+    "spaceBetweenLabelSwitch": "8px"
+  },
+  "tag": {
```
[the remark this answers is not in the document]
Comment (GomezIvann) [reply website/screens/theme-generator/themes/advanced-theme.json:834]: Agreed, done!

**Review by raquelarrojo — APPROVED** [review]

**Pull request merged**